package.json

```diff
@@ -21,6 +21,9 @@
   "engines": {
     "node": ">=6.0.0"
   },
+  "files": [
+    "lib"
```

Comment:
Need to include `index.js`

Reply:
Nope, the `mail` file is automatically added :)

Reply:
```
$ npm pack
npm notice
npm notice package: sequelize@5.0.0-beta.14
npm notice === Tarball Contents ===
npm notice 6.2kB   package.json
npm notice 116B    index.js
...
```